src/install/unix.rst

```diff
@@ -229,10 +233,66 @@ CouchDB no longer ships with any daemonization scripts.
 
 The couchdb team recommends `runit <http://smarden.org/runit/>`_ to
 run CouchDB persistently and reliably. Configuration of runit is
-straightforward; if you have questions, reach out to the CouchDB
-user mailing list.
+straightforward; if you have questions, contact the CouchDB
+`user mailing list <http://mail-archives.apache.org/mod_mbox/couchdb-user/>`_
+or `IRC-channel #couchdb <http://webchat.freenode.net/?channels=#couchdb>`_
+in FreeNode network.
+
+Let's consider configuring runit on Ubuntu 16.04. The following
```

Comment:
Too informal, the paragraph should be tightened to explain what runit is going to do that's valuable, and then how to configure it.